test/transforms/dec128.test.js

```diff
@@ -22,52 +24,101 @@ const doesNotChangeNumbers = {
     "(1 + (4 - 2)) * (3 + 4);",
 };
 
+const constructor = {
```

Comment:
Someday it might be nice to make a `shared.js` test file and call them with the `libName` in the named files, but I didn't need to make this one any bigger.